src/helpers.js

```diff
@@ -8,6 +8,7 @@ import cryptoRandomString from 'crypto-random-string'
 // eslint-disable-next-line import/no-extraneous-dependencies, import/extensions
 import { Range } from 'atom'
 
+const lintRules = new Map()
```

Comment:
It feels to me like if we go this route, `lintRules` should be a property on a `class`, and `updateFixableRules`, `updateRules`, and `getRules` should be methods on that class.  Normally I'd advocate for a more functional style, but I find maps to be difficult to work with in pure functions.